extensions/amp-access/0.1/amp-login-done-dialog.js

```diff
@@ -239,10 +239,22 @@ export function buildLangSelector(lang) {
       langPrefix += '-';
     }
     langPrefix += i == 0 ? parts[i].toLowerCase() : parts[i].toUpperCase();
-    if (langPrefix.indexOf('"') != -1) {
-      langPrefix = langPrefix.replace(/\"/g, '');
-    }
+    langPrefix = langPrefix.replace(/[^a-zA-Z\-]/g, '');
     langExpr += `[lang="${langPrefix}"]`;
   }
   return langExpr;
 }
+
+
+/**
+ * Ensures the provided URL is an absolute http/https URL.
+ * @param {string} url
+ * @return {string}
+ * @private Visible for testing.
+ */
+export function assertAbsoluteUrl(url) {
```

Comment:
I thought about this, but a bit of a concern is that people are likely to test on a non-local http. We normally allow all development to proceed on non-local http and this would be one of the few things that we will not allow this way. wdyt?